packages/features/bookings/Booker/types.ts

```diff
@@ -42,5 +44,5 @@ export interface BookerProps {
 }
 
 export type BookerState = "loading" | "selecting_date" | "selecting_time" | "booking";
-export type BookerLayout = "small_calendar" | "large_timeslots" | "large_calendar" | "mobile";
+export type BookerLayout = BookerLayouts | "mobile";
```

Comment:
Reuse types here. Mobile isn't a valid setting for the end user, but is a layout that the component itself uses. That's why we add that still here.